core/source/api/opencmiss/zinc/fieldimage.h

```diff
@@ -481,14 +481,74 @@ ZINC_API int cmzn_field_image_set_wrap_mode(cmzn_field_image_id image_field,
  * Gets the property named in the given property string from the given field image.
  * The returned string must be deallocated by the receiver.
  *
- * @param	image   	The image field.
+ * @param	image_field   	The image field.
  * @param	property	The property to retrieve.
  *
  * @return	null if it fails, else an allocated string containing the properties value.
  */
-ZINC_API char *cmzn_field_image_get_property(cmzn_field_image_id image,
+ZINC_API char *cmzn_field_image_get_property(cmzn_field_image_id image_field,
 	const char* property);
 
+/**
+ * Get the number of bytes per component of this image field.
+ *
+ * @param image_field 	The image field.
+ *
+ * @return  number of bytes of this image field.
+ */
+ZINC_API int cmzn_field_image_get_number_of_bytes_per_component(
```

Comment:
I think this should be number of bits per component with only 8 or 16 currently supported - then it's consistent with CMZN_STREAMINFORMATION_IMAGE_ATTRIBUTE_BITS_PER_COMPONENT. It also allows us to support 12 bit, 1 bit etc. if needed in future.
Likewise for setter below.
BTW good you have used direct API rather than get/set generic attributes which I'm no longer in favour of.

Reply:
Sure.